prometheus/README.md

```diff
@@ -0,0 +1,68 @@
+# Prometheus
```

Comment:
This looks great.  I think it would be better if you add a section where you show a query using the data.  e.g. aggregate by minute or something.

Reply:
Sounds good! What's the best way to deliver that in a sample app? Maybe a standalone python file that users can copy-paste into the UI?

Reply:
I think that would work well, using the panel data just put in the readme?

Reply:
I did some tinkering and I think the best way to do this would be an additional method within the prometheus.py script. If there's a better way I'd love to try it out